### PR TITLE
Zipkin: Allow SetTag even when the CS annotation is missing (#1414)

### PR DESCRIPTION
**Pull request by clguimanMSFT**
*  Removed the check for CS annotation in ZipkinSpan::SetTag
*  Removed  ZipkinSpan::hasCSAnnotation (SetTag was the only place where it was used)
*  Updated Zipkin Tracer unit tests

### CODE REVIEW

**Comment by mattklein123** [comment]
@rshriram @fabolive please review.

cc @adriancole if you have time for review.

**Comment by fabolive** [comment]
@mattklein123 I will review this later today.

**Comment by RomanDzhabarov** [comment]
LGTM, will defer final review to @fabolive

**Comment by clguimanMSFT** [comment]
I had a compilation error in zipkin_tracer_impl_test (I missed it due to other tests failing locally), so I amended the commit

**Comment by mattklein123** [comment]
@clguimanMSFT thanks in the future once you open PRs please don't rebase (just push new commits). It's easier to review. Thank you.

File: ci/bazel/get_workspace_status [hunk 1]
```diff
@@ -0,0 +1 @@
+/source/bazel/get_workspace_status
```
Comment (mattklein123) [inline ci/bazel/get_workspace_status]: Can you revert this change and below?

Comment (clguimanMSFT) [inline ci/bazel/get_workspace_status]: sorry for that, I have reverted the change

**Comment by clguimanMSFT** [comment]
Is there some easy way I can just run specific tests? Currently I have a lot of test failures locally and I would like to run just the tests directly affected by my change.

**Comment by mattklein123** [comment]
If you are using the CI container directly, IIRC we don't have an option to override the tests. You could hack the script that launches the tests, for example: https://github.com/lyft/envoy/blob/master/ci/do_ci.sh#L49

**Comment by mattklein123** [comment]
@clguimanMSFT this is passing tests now. Are you OK to commit or did you want to look at something else?

**Comment by clguimanMSFT** [comment]
@mattklein123 I'm OK with the commit.
Thanks for the help
